### PR TITLE
save and monitor disk space

### PR DESCRIPTION
**Pull request by jkrauska**
- changes a few cp to mv 
- removes temporary tars
- checks disk space a few times during run

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -120,7 +122,9 @@ jobs:
                     - homebrew-v5-
             - run:
                 name: Install macos dependancies - homebrew - make macos-setup-download
```
Comment (psteckler) [inline .circleci/config.yml]: "dependencies" misspelled throughout (sorry, I notice these things)

Comment (jkrauska) [reply .circleci/config.yml]: den it is

File: scripts/macos-setup-opam.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+if [ "$SWITCH_FOUND" = true ]; then
+  opam switch set $SWITCH
+else
+  opam switch create $SWITCH || true
+  opam switch $SWITCH
+fi
+
+# Test -- see if curent export is different from saved export
+opam switch export opam.export.test
+diff -U1 opam.export.test src/opam.export || true
```
Comment (bkase) [inline scripts/macos-setup-opam.sh]: how will we remember to update this?

Comment (jkrauska) [reply scripts/macos-setup-opam.sh]: this is a debug/wip

I'm trying to understand if import is needed or not by looking at this diff.

I'm still not sure if import ALWAYS reinstalls or just does a delta apply.

**Comment by jkrauska** [comment]
it worked -- merging @bkase

**Pull request merged**